source/_integrations/pid_controller.markdown

```diff
@@ -0,0 +1,152 @@
+---
+title: PID Controller
+description: Creates a PID controller to regulate a sensor value with a number-entity output.
+ha_category:
+  - Number
+ha_release: '2023.05'
+ha_iot_class: Local Polling
+ha_domain: pid_controller
+ha_platforms:
+  - number
+ha_integration_type: integration
+---
+
+The `pid_controller` platform is a `number`. It can be used to control a number entity output to regulate a sensor value to a certain setpoint. The value of the number entity is the setpoint. As a sensor, any numerical senosor entity can be used. If two sensors are configured, the PID controller will act as a differential controller, using the difference between the two sensor values as input signal.
```

Comment:
```suggestion
The `pid_controller` platform is a `number`. It can be used to control a number entity output to regulate a sensor value to a specific setpoint. The value of the number entity is the setpoint. As a sensor, any numerical sensor entity can be used. If two sensors are configured, the PID controller will act as a differential controller, using the difference between the two sensor values as input signal.
```